lib/Sema/TypeCheckConcurrency.cpp

```diff
@@ -637,7 +640,8 @@ void swift::checkActorIsolation(const Expr *expr, const DeclContext *dc) {
         Expr *fn = call->getFn()->getValueProvidingExpr();
         if (auto memberRef = findMemberReference(fn)) {
           checkMemberReference(
-              call->getArg(), memberRef->first, memberRef->second);
+              call->getArg(), memberRef->first, memberRef->second, 
+              /*isEscapingPartialApply=*/false, call);
```

Comment:
I think the same "calling a synchronous function as async" behavior works for global actors as well, which would be under a general `ApplyExpr` rather than only the `SelfApplyExpr` above.

Reply:
👍 took care of this